draft-packages/tag/docs/Tag.stories.tsx

```diff
@@ -31,117 +23,139 @@ export default {
   decorators: [withDesign],
 }
 
-export const DefaultMediumKaizenSiteDemo = args => (
-  <StoryContainer>
-    <Tag variant="default" {...args}>
-      Default
-    </Tag>
-  </StoryContainer>
-)
-DefaultMediumKaizenSiteDemo.storyName = "Default - Medium (Kaizen Site Demo)"
-
-export const Status = () => (
-  <StoryContainer>
-    <StorySection>
-      <Tag variant="default" size="small">
-        Default
-      </Tag>
-      <Tag variant="default" size="medium">
-        Default
-      </Tag>
-    </StorySection>
-    <StorySection>
-      <Tag variant="statusLive" size="small">
-        Live
-      </Tag>
-      <Tag variant="statusLive" size="medium">
-        Live
-      </Tag>
-    </StorySection>
-    <StorySection>
-      <Tag variant="statusDraft" size="small">
-        Draft
-      </Tag>
-      <Tag variant="statusDraft" size="medium">
-        Draft
-      </Tag>
-    </StorySection>
-  </StoryContainer>
+export const DefaultStory = args => (
+  <Tag variant="default" {...args}>
+    Default
+  </Tag>
 )
-Status.storyName = "Status"
-Status.parameters = { chromatic: { disable: false } }
+DefaultStory.storyName = "Tag"
+DefaultStory.parameters = { chromatic: { disable: false } }
```

Comment:
Oops sorry I missed this. We don't need to send the default story to Chromatic :)